tests/dockerConfiguration/verifySrvDataDirectory_test.js

```diff
@@ -4,50 +4,57 @@ const moment = require('moment');
 Feature('Test PMM server with srv volume and password enw variable');
 
 let testCaseName = '';
-const dockerVersion = process.env.DOCKER_VERSION || 'perconalab/pmm-server:dev-latest';
+const dockerVersion = process.env.DOCKER_VERSION || 'perconalab/pmm-server:3-dev-latest';
 
 const runContainerWithoutDataContainer = async (I) => {
-  await I.verifyCommand(`docker run -v $HOME/srvNoData:/srv -d --restart always --publish 8081:80 --name pmm-server-srv ${dockerVersion}`);
+  await I.verifyCommand('mkdir $HOME/srvNoData/ || true');
+  await I.verifyCommand('chmod -R 777 $HOME/srvNoData/ || true');
```

Comment:
Is there a reason why the access permissions are not set to `pmm:pmm`?

See migration guide - step 2 - https://docs.percona.com/percona-monitoring-and-management/3/pmm-upgrade/migrating_from_pmm_2.html#step-2-migrate-pmm-2-server-to-pmm-3

Reply:
User pmm:pmm is not yet created sice this is a new container. when using chown pmm:pmm i get error that user is not existing. This works ok